### PR TITLE
Update image names on charts

### LINKED ISSUE #44: Update image names on charts
Chart image names are wrong after ghcr update on #40

### PR DESCRIPTION
**Pull request by khaosdoctor**
_Image names are wrong after #40 on charts_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #44

### CODE REVIEW

**Comment by khaosdoctor** [comment]
Add @arschles for review

**Comment by arschles** [comment]
Looks good @khaosdoctor . gonna wait until we get #40 merged to do this

**Review by arschles — CHANGES_REQUESTED** [review]
@khaosdoctor the images names will need to use the `:latest` tag. I know you're out today, so I'll open a new PR with that change in it, close this one, and credit you in the new one - so we can do the release.

File: charts/keda-http-operator/values.yaml
Comment (tomkerkhove) [inline charts/keda-http-operator/values.yaml:4]: It's a nit, but it's best to use some structure and be more consistent:
```suggestion
images:
  operator: ghcr.io/kedacore/http-add-on-operator
  interceptor: ghcr.io/kedacore/http-add-on-interceptor
  externalScaler: ghcr.io/kedacore/http-add-on-scaler
```

You'll have to update all references though.

**Pull request closed without merging**
